graphdatascience/query_runner/gds_arrow_client.py

```diff
@@ -99,6 +101,7 @@ def __init__(
         tls_root_certs: Optional[bytes] = None,
         arrow_endpoint_version: ArrowEndpointVersion = ArrowEndpointVersion.V1,
         user_agent: Optional[str] = None,
+        arrow_client_options: Optional[dict[str, Any]] = None,
```

Comment:
you could also pass only **kwargs here and pass these to the dict.
As its the gds-arrow-client it feels a bit redundant.